packages/icons/src/components/spinners/spinners.module.scss

```diff
@@ -0,0 +1,143 @@
+@import '@tpr/theming/lib/variables.scss';
+
+.wrapper1 {
```

Comment:
Can we use more descriptive names for these classes to make it easier to choose the required effect?

Reply:
classes have been renamed